File: Practical03/1.py
n1=int(input("Enter n1 value : "));
n2=int(input("Enter n2 value : "));

if(n1<n2):
    for i in range(n1,n2+1):
        if(i%2!=0):
            print(i)
else:
    #print("N1 must be greater then n2")
    i=n2
    while(i>=n2):
        print(i)
    i=i-1

File: php5/MA019/Practical_4/1.php
<!DOCTYPE html>
<html>
<head>
	<title></title>
</head>
<body>
	<form method="GET" action="1.php">
	<table border="1">
		<tr><td>Enter number</td>
			<td><input type="text" name="text"><td><button>Submit</button></td></tr>
<?php
	if(isset($_GET['text']))
	{
		$number=$_GET['text'];
		$r=strrev($number);
		if($r==$number)
		{
			echo "the number ".$number." is palindrome";
		}
		else
		{
			echo "the number ". $number ." is not palindrome";
		}
	}
?>
</table>
</form>
</body>
</html>
